frontend/src/employee-frontend/components/reports/AssistanceNeedsAndActions.tsx

```diff
@@ -329,11 +308,6 @@ function AssistanceNeedsAndActions() {
                   <Td>
                     {reducePropertySum(filteredRows, (r) => r.otherBasisCount)}
                   </Td>
-                  {featureFlags.assistanceBasisOtherEnabled && (
```

Comment:
This removes the wrong column because the feature flag has been originally put in the wrong column :see_no_evil:

Reply:
Yeah, and the query was also a bit wrong. The report now works after the latest push.